loader_hub/confluence/base.py

```diff
@@ -92,13 +106,26 @@ def load_data(self, space_key: Optional[str] = None, page_ids: Optional[List[str
                 docs.append(doc)
 
         if page_ids:
+            # with the include children option we will dfs and get the children of all the pages 
+            # requested
+            if include_children:
+                page_ids = self._dfs_page(self.confluence, page_ids[0])
             for page_id in page_ids:
                 page = self.confluence.get_page_by_id(page_id=page_id, expand='body.storage.value')
                 doc = self.process_page(page, include_attachments, text_maker)
                 docs.append(doc)
 
         return docs
 
+    def _dfs_page(self, raw_confluence, page_id):
```

Comment:
I can combine this and the getting the page itself as well but keeping it this way for now.